tests/Constraint/MultiConstraintTest.php

```diff
@@ -339,92 +339,35 @@ public function multiConstraintOptimizations()
                 '^1.0 || ^2.0 !=2.0.1 || ^3.0 || ^4.0',
                 new MultiConstraint(
                     array(
-                        new MultiConstraint(
-                            array(
-                                new Constraint('>=', '1.0.0.0-dev'),
-                                new Constraint('<', '2.0.0.0-dev'),
-                            ),
-                            true // conjunctive
-                        ),
-                        new MultiConstraint(
-                            array(
-                                new Constraint('>=', '2.0.0.0-dev'),
-                                new Constraint('<', '3.0.0.0-dev'),
-                                new Constraint('!=', '2.0.1.0'),
-                            ),
-                            true // conjunctive
-                        ),
-                        new MultiConstraint(
-                            array(
-                                new Constraint('>=', '3.0.0.0-dev'),
-                                new Constraint('<', '5.0.0.0-dev'),
-                            ),
-                            true // conjunctive
-                        ),
+                        new Constraint('>=', '1.0.0.0-dev'),
+                        new Constraint('<', '5.0.0.0-dev'),
+                        new Constraint('!=', '2.0.1.0'),
                     ),
-                    false
+                    true
                 )
             ),
             'Test does not collapse multiple contiguous with multiple other constraint' => array(
```

Comment:
the description of these test cases should probably be updated

Reply:
lazy me !

thanks